theme/less/main.less

```diff
@@ -4,20 +4,46 @@
 body
 {
   margin: 0px;
-  font-family: sans-serif;
+  font-family: "PT Sans", "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif;
 }
 
 header
 {
   background-color: @primary;
   padding: 30px;
+  
 }
 
 header > div
 {
   margin: 0px auto;
   max-width: 1000px;
   color: @reverse;
+  
+}
+
+header > div > h3
+{
+  color: #7d7d7d;
+  font-size: 24px;
+  letter-spacing: -1px;
+  
+}
+
+header > div > h1, header > div > h1 a
+{
+  color: #262626;
+  font-size: 44px;
+  text-decoration: none;
+  font-weight: bold;
+  line-height: 1px;
+  letter-spacing: -2px;
+  text-shadow: 0px 1px 0px #FFF;
+}
+
+header > div > h1 a:hover
+{
+  color:#3253ad;
```

Comment:
Suggestion: Let's simplify selectors here, you could use `header h3`, `header  h1` and `header  h1 a` instead.

Reply:
OK